content/zh/docs/setup/platform-setup/k3d/index.md

```diff
@@ -0,0 +1,133 @@
+    {{< /text >}}
+
+    {{< tip >}}
+    `k3d-` is prefixed to the context and cluster names, for example: `k3d-k3s-default`
+    {{< /tip >}}
+
+1. 如果运行多个集群，则需要选择 `kubectl` 与哪个集群进行对话。您可以设置默认集群通过在 [Kubernetes kubeconfig](https://kubernetes.io/zh-cn/docs/concepts/configuration/organize-cluster-access-kubeconfig/) 文件中设置当前上下文来实现 `kubectl`。此外，您可以运行以下命令为 `kubectl` 设置当前上下文。
+
+    {{< text bash >}}
+    $ kubectl config use-context k3d-k3s-default
+    Switched to context "k3d-k3s-default".
+    {{< /text >}}
+
+## 为 k3d 设置 Istio
+
+1. 完成 k3d 集群的设置后，可以继续在其上 [使用 Helm 3安装 Istio](/zh/docs/setup/install/helm/)。
+
+    {{< text bash >}}
+    $ kubectl create namespace istio-system
+    $ helm install istio-base istio/base -n istio-system --wait
+    $ helm install istiod istio/istiod -n istio-system --wait
+    {{< /text >}}
+
+1. (可选) 安装入口网关:
+
+    {{< text bash >}}
+    $ kubectl label namespace istio-system istio-injection=enabled
+    $ helm install istio-ingressgateway istio/gateway -n istio-system --wait
+    {{< /text >}}
+
+## 为 k3d 设置仪表板用户界面
+
+k3d 没有像 minikube 这样的内置仪表板 UI。但是您仍然可以设置 Dashboard (基于 web 的 Kubernetes UI) 来查看您的集群。
+按照以下说明为 k3d 设置仪表板。
+
+1. 要部署仪表板，请运行以下命令:
+
+    {{< text bash >}}
+    $ GITHUB_URL=https://github.com/kubernetes/dashboard/releases
+    $ VERSION_KUBE_DASHBOARD=$(curl -w '%{url_effective}' -I -L -s -S ${GITHUB_URL}/latest -o /dev/null | sed -e 's|.*/||')
+    $ kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/${VERSION_KUBE_DASHBOARD}/aio/deploy/recommended.yaml
+    {{< /text >}}
+
+1. 验证仪表板已部署并正在运行。
+
+    {{< text bash >}}
+    $ kubectl get pod -n kubernetes-dashboard
+    NAME                                         READY   STATUS    RESTARTS   AGE
+    dashboard-metrics-scraper-8c47d4b5d-dd2ks    1/1     Running   0          25s
+    kubernetes-dashboard-67bd8fc546-4xfmm        1/1     Running   0          25s
+    {{< /text >}}
+
+1. 创建 `serviceaccount` 和 `clusterrolebinding` 为新创建的集群提供管理员访问权限。
+
+    {{< text bash >}}
+    $ kubectl create serviceaccount -n kubernetes-dashboard admin-user
+    $ kubectl create clusterrolebinding -n kubernetes-dashboard admin-user --clusterrole cluster-admin --serviceaccount=kubernetes-dashboard:admin-user
+    {{< /text >}}
+
+1. 要登录到您的仪表板，您需要一个承载令牌。使用以下命令将令牌存储在变量中。
+
+    {{< text bash >}}
+    $ token=$(kubectl -n kubernetes-dashboard create token admin-user)
+    {{< /text >}}
+
+    使用 `echo` 命令显示令牌，并将其复制以用于登录到仪表板。
+
+    {{< text bash >}}
+    $ echo $token
+    {{< /text >}}
+
+1. 您可以通过运行以下命令使用 kubectl 命令行工具访问仪表板:
+
+    {{< text bash >}}
+    $ kubectl proxy
+    Starting to serve on 127.0.0.1:8001
+    {{< /text >}}
+
+    单击 [Kubernetes 仪表板](http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/) 来查看您的部署和服务。
+
+    {{< warning >}}
+    You have to save your token somewhere, otherwise you have to run step number 4 everytime you need a token to log in to your Dashboard.
```

Comment:
```suggestion
    You have to save your token somewhere, otherwise you have to run step number 4 everytime you need a token to log in to your Dashboard.
```
这句话需要翻译